incubator/jenkins/master-image/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM jenkins:2.7.3
```

Comment:
+1 On the self contained chart including images. I'm with @viglesiasce.  IMHO it provides more transparency and visiblity